Description: Use yaml.v2 library instead of yaml.v1
 The yaml.v1 library is unpackage, obsolete and can be replaced by
 yaml.v2 in most circumstances.
Author: Tim Potter <tpot@hpe.com>
Forwarded: No
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: golang-github-coreos-pkg/yamlutil/yaml.go
===================================================================
--- golang-github-coreos-pkg.orig/yamlutil/yaml.go
+++ golang-github-coreos-pkg/yamlutil/yaml.go
@@ -5,7 +5,7 @@ import (
 	"fmt"
 	"strings"
 
-	"gopkg.in/yaml.v1"
+	"gopkg.in/yaml.v2"
 )
 
 // SetFlagsFromYaml goes through all registered flags in the given flagset,
